microsoft/testsuites/display/drm_sanity.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, cast
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+
+from lisa.base_tools import Cat
+from lisa.tools import Lsmod
+
+@TestSuiteMetadata(
+    area="drm",
+    category="functional",
+    description="""
+    This test suite uses to verify drm driver sanity.
+    """,
+    requirement=simple_requirement(unsupported_os=[]),
+)
+class DRM_Sanity(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This case is to check whether the drm driver registered successfully.
+
+        Step,
+        1. Check hyperv_drm driver is inserted correctly.
+        """,
+        priority=0,
```

Comment:
By default, all test cases should be P2. There is only one P0 test case, it measures test case can boot up or not. Let me know the impact of the failure, we can decide if it needs to be P1.